services/user-svc/internal/hwkc/models.go

```diff
@@ -0,0 +1,28 @@
+package hwkc
+
+type User struct {
+	ID *string `json:"id,omitempty"`
+}
+
+type Organization struct {
+	ID            *string              `json:"id,omitempty"`
+	Name          *string              `json:"name,omitempty"`
+	DisplayName   *string              `json:"displayName,omitempty"`
+	RawAttributes *map[string][]string `json:"attributes,omitempty"`
```

Comment:
make this private and a proper struct (`struct IsPersonal []string ...`), which is unmarshalled as expected (but as a []string) and add a method `(o *Organization) IsPersonal() bool` which reads out that value similar to the current ParseRawAttributes(), which will then be thrown out

Reply:
Very good idea. Thanks. For the unmarshall process, `Attributes` needs to be public.